cabal.project

```diff
@@ -4,6 +4,9 @@
 --
 -- export PATH=$PWD/dist-newstyle/build/x86_64-linux/ghc-7.10.3/hpb-0.1.1/c/hpb/build/hpb:$PATH
 
+-- See https://github.com/Happstack/boomerang/issues/12
```

Comment:
Do you expect that this issue will be resolved soon? If so, it would be nice to wait until that makes it to Hackage so that we don't have to add `allow-newer`s to the `cabal.project` files of `crucible` and its dependencies.

Reply:
> Do you expect that this issue will be resolved soon?

I don't have strong enough basis for judgment to make a prediction about this either way. I think the relevant data are basically:

- The last commit to `boomerang` was 11 months ago.
- The maintainer responded to my PR within a few days.

I'm fine waiting for a bit, though I worry about merge conflicts. Perhaps we can set a date to check back in?

Reply:
Sure. Are you content with waiting until the end of this week?

Reply:
Definitely!

Reply:
Alright, what do we want to do here? There has been no further response from the maintainer.

Reply:
In that case, I am fine with moving forward with this, although we should be mindful of this dependency going forward. The only reason we incur a `boomerang` dependency in the first place is due to `itanium-abi`, and [removing the `itanium-abi` dependency](https://github.com/GaloisInc/crucible/issues/1235) would be one way to solve this problem. Alternatively, we can periodically check in on https://github.com/Happstack/boomerang/issues/12 and remove this `allow-newer` once the changes have made it to Hackage.